File: src/main/java/org/noxet/noxetserver/messaging/SuccessMessage.java
package org.noxet.noxetserver.messaging;

import net.md_5.bungee.api.ChatColor;

public class SuccessMessage extends Message {
    public SuccessMessage(String text) {
        super(text);
    }

    @Override
    public ChatColor getDefaultColor() {
        return ChatColor.GREEN;
    }

    @Override
    public String getPrefix() {
        return super.getPrefix() + "§2Success: ";
    }
}

File: src/main/java/org/noxet/noxetserver/messaging/SuccessMessage.scala
package org.noxet.noxetserver.messaging;

import net.md_5.bungee.api.ChatColor;

class SuccessMessage(text: String) extends Message(text):
    override def getDefaultColor: ChatColor = ChatColor.GREEN
    override def getPrefix: String = super.getPrefix + "§2Success: "
    